travharv/store.py

```diff
@@ -40,8 +40,19 @@ def select_subjects(self, sparql) -> List[str]:
         # todo convert response into list of subjects
         list_of_subjects = [row[0] for row in result]
         log.debug(f"length list_of_subjects: {len(list_of_subjects)}")
+        log.debug(f"list_of_subjects: {list_of_subjects}")
         return list_of_subjects
 
+    def select_subjects_for_ppath(self, subject, property_path, NSM):
+        pre_sparql = self._qryBuilder.build_syntax(
+            "trajectory.sparql",
+            subject=subject,
+            property_trajectory=property_path,
+        )
+        sparql = resolve_sparql(pre_sparql, NSM)
```

Comment:
when we move to 004 one single project it would be easy/ logical to consider optionally passing the NSm to the rdfj2 querybuilder to already cater for the resolve part so this can easily become one step operation...